sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/module-info.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+module com.azure.monitor.opentelemetry.autoconfigure {
+
+    requires transitive com.azure.core;
+
+    requires java.management;
+
+    requires io.netty.handler;
```

Comment:
Long term we should understand why this requires this as this means we are tying into Netty somewhere.